api/api_public_blockchain.go

```diff
@@ -63,13 +63,23 @@ func (s *PublicBlockChainAPI) BlockNumber() *big.Int {
 }
 
 // ChainID returns the chain ID of the chain from genesis file.
+// This will be deprecated.
```

Comment:
Do we need to deprecate this? Some users may use this method.

Reply:
Right. We don't need to remove it. But I want let users know this is old one. ^^

Reply:
I understand your point, but the comment little confusing me. One of the same methods "will be deprecated", and the other is "for compatibility". It's hard to distinguish which one is the main method. I think one of the two comments ("This will be deprecated", "This is for compatibility with ethereum client") is enough.

Reply:
Got it. I removed 1st one.